src/test/java/org/jenkinsci/plugins/secure_requester_whitelist/SecureRequesterImplTest.java

```diff
@@ -39,10 +39,13 @@
     @Rule public JenkinsRule r = new JenkinsRule();
 
     @PresetData(PresetData.DataSet.NO_ANONYMOUS_READACCESS)
-    @Test public void authorizing() throws Exception {
+    @Test
```

Comment:
I am reverting this one to make it consistent with the style in the rest of project.